CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(deep_trekker VERSION 0.0)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Let's not do these things without discussing it first ?

We really need to have a process to choose C++ standard (beyond what our dependencies require). Moreover, there is the small-ish issue that orogen itself does not have support to specify it, to parse the headers.

Reply:
Ah, I knew it had to be discussed first. I guess I tried adding it while I was trying things out to fix that bug with the curlpp + orogen interaction and forgot about it.